docs/source/conf.py

```diff
@@ -279,6 +279,14 @@
 # If false, no module index is generated.
 #latex_domain_indices = True
 
+# this is to avoid the error
+# aiida/orm/implementation/django/querybuilder.py:docstring of
+# aiida.orm.implementation.django.querybuilder.array_length._compiler_dispatch:1: WARNING: Unknown target name: "visit".
```

Comment:
weird!
do we know what's so special about this code?

Reply:
I know, Maybe something to-do with the `_` 🤷 https://github.com/sphinx-doc/sphinx/issues/4350, but that docs are inherited comes from sqlachemy anyway (I think), so we can't really "fix" it